spk/memcached/src/service-setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+# Package
+PACKAGE="memcached"
+
+# Others
+INSTALL_DIR="/var/packages/${PACKAGE}/target/"
```

Comment:
please use system provided `SYNOPKG_PKGDEST` variable in this script (`INSTALL_DIR` is the very old way we did it in install.sh scripts).

dito for `PACKAGE` variable. If you really need the plain package name, please  use `SYNOPKG_PKGNAME` instead.